### PR TITLE
Minor changes to setup.py

### PR DESCRIPTION
**Pull request by smaranjitghose**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,22 +17,36 @@
 from setuptools import setup, find_packages
 from os import path
 from io import open
+from setuptools import find_packages, setup
```
Comment (hastagAB) [inline setup.py:20]: We already imported `find_packages` and `setup` on line number 17

File: setup.py [hunk 2]
```diff
@@ -17,22 +17,36 @@
...
 
 here = path.abspath(path.dirname(__file__))
+
 # fetch the long description from the README.md
 with open(path.join(here, 'README.md'), encoding='utf-8') as f:
     long_description = f.read()
+    
+# Read the requirements
```
Comment (hastagAB) [inline setup.py:28]: Nirjas is written completely in python and without any third-party dependency. So we don't really need a `requirements.txt` file.

File: setup.py [hunk 3]
```diff
@@ -17,22 +17,36 @@
...
+with (here / "requirements.txt").open(encoding="utf8") as f:
+    requirements = f.readlines()
 
 
 CLASSIFIERS = """\
 Development Status :: 4 - Beta
+Topic :: Software Development :: Build Tools
```
Comment (hastagAB) [inline setup.py:35]: We do not associate ourselves as a build tool, we have a very specific use case. perhaps have a look at the [README](https://github.com/fossology/nirjas#description) to know more.

File: setup.py [hunk 4]
```diff
@@ -45,14 +59,13 @@
     url='https://github.com/fossology/nirjas',  
     author='Ayush Bhardwaj, Kaushlendra Pratap',  
     author_email='classicayush@gmail.com, kaushlendrapratap.9837@gmail.com',  
-
     classifiers=[_f for _f in CLASSIFIERS.split('\n') if _f],
-    keywords='Comment Extractor, Code Comment Extractor, Source Code Extractor, Source Extractor',  
+    keywords='Comment Extractor, Code Comment Extractor, Source Code Extractor, Source Extractor, Code Snippet Separator',
```
Comment (hastagAB) [inline setup.py:63]: Can you explain this keyword? We don't separate any specific code snippet rather we extract the whole source code from a file. This is the reason we emphasized on the word `Extraction`.
Please have a look at our [README](https://github.com/fossology/nirjas#description) to know more.

File: setup.py (hunk 1 above)
Comment (smaranjitghose) [reply setup.py:20]: Okay, my bad

**Pull request closed without merging**